### PR TITLE
made changes

### PR DESCRIPTION
**Pull request by ChrisDietrich405**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import "./App.css";
 function App() {
   const [count, setCount] = useState(0);
 
-  return <h1>FROM PULL REQUEST</h1>;
```
Comment (jesusebzcp) [inline src/App.jsx:8]: No Very good (example 👀)

Comment (jesusebzcp) [reply src/App.jsx:8]: @ChrisDietrich405

Comment (jesusebzcp) [reply src/App.jsx:8]: i dot not agree @ChrisDietrich405 🚀

Comment (ChrisDietrich405) [reply src/App.jsx:8]: I don't understand, I built the button to the tech lead's specifications

Comment (ChrisDietrich405) [reply src/App.jsx:8]: I'm not going to fire you

**Review by jesusebzcp — APPROVED** [review]

**Comment by jesusebzcp** [comment]
All Good

**Pull request merged**